js/views/conversation_view.js

```diff
@@ -1007,7 +1007,7 @@
       this.listenBack(view);
     },
 
-    showContactDetail(contact) {
+    showContactDetail({ contact, hasSignalAccount }) {
```

Comment:
I see the function signature has changed here. Should there be a corresponding change to the call site? Or was that already done before and this adapts to it?

Reply:
The `.trigger()` call below is what sends parameters to this function, via Backbone eventing.